x/ibc/07-tendermint/types/errors.go

```diff
@@ -10,11 +10,13 @@ const (
 
 // IBC tendermint client sentinel errors
 var (
-	ErrInvalidTrustingPeriod  = sdkerrors.Register(SubModuleName, 2, "invalid trusting period")
-	ErrInvalidUnbondingPeriod = sdkerrors.Register(SubModuleName, 3, "invalid unbonding period")
-	ErrInvalidHeader          = sdkerrors.Register(SubModuleName, 4, "invalid header")
-	ErrInvalidMaxClockDrift   = sdkerrors.Register(SubModuleName, 5, "invalid max clock drift")
-	ErrTrustingPeriodExpired  = sdkerrors.Register(SubModuleName, 6, "time since latest trusted state has passed the trusting period")
-	ErrUnbondingPeriodExpired = sdkerrors.Register(SubModuleName, 7, "time since latest trusted state has passed the unbonding period")
-	ErrInvalidProofSpecs      = sdkerrors.Register(SubModuleName, 8, "invalid proof specs")
+	ErrInvalidChainID         = sdkerrors.Register(SubModuleName, 2, "invalid chain-id")
+	ErrInvalidTrustingPeriod  = sdkerrors.Register(SubModuleName, 3, "invalid trusting period")
+	ErrInvalidUnbondingPeriod = sdkerrors.Register(SubModuleName, 4, "invalid unbonding period")
+	ErrInvalidHeight          = sdkerrors.Register(SubModuleName, 5, "invalid height")
```

Comment:
rename to invalid header height? it is nice when the base error msg has some contextual information, height is a generic reference